server/sonar-server/src/main/java/org/sonar/server/computation/period/PeriodsHolder.java

```diff
@@ -32,6 +32,8 @@
  */
 public interface PeriodsHolder {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move constants to a class or enum. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1214)